File: GoLang/6 kyu/Multiplication table/main.go
package main

import "fmt"

func createRow(count, num int) []int {
	var row []int
	temp := count
	for i := 0; i < num; i++ {
	  row = append(row, count)
	  count += temp
	}
	return row
}

func MultiplicationTable(size int) [][]int {
	var table [][]int
	for i := 1; i <= size; i++ {
		table = append(table, createRow(i, size))
	  }
	return table
}

func main() {
	fmt.Println(MultiplicationTable(5))
}

File: Python/6 kyu/Multiplication table/main.py
def multiplication_table(size):
    table = []
    for i in range(1, size + 1):
        row = []
        for k in range(i, i*size+1):
            if k%i == 0:
                row.append(k)
        table.append(row)
    return table

print(multiplication_table(5))